src/MainContext/MainContext.types.ts

```diff
@@ -0,0 +1,20 @@
+export enum CartActionTypes {
```

Comment:
![IMG_0009](https://github.com/fs-jan24-CodeCrafters/fs-jan24-CodeCrafters/assets/130569769/f4230678-bdb9-49a9-942c-dd1426227ca1)

You can make your type check for actions like that and then you don’t need to use separate emus for  actions names.